src/ui/Pagination.tsx

```diff
@@ -65,7 +65,9 @@ const Pagination = ({
           border-color: ${getValue('borderColor')};
           padding: ${getValue('paddingY')} ${getValue('paddingX')};
 
-          &:hover {
+          border-radius: 8px;
+
+          */ &:hover {
```

Comment:
Should this be part be in comment? 
Or can it be removed?

```suggestion
           &:hover {
```